src/Link_TEST.cc

```diff
@@ -203,3 +205,61 @@ TEST(DOMLink, InvalidInertia)
   EXPECT_DOUBLE_EQ(3.4, link.Inertial().MassMatrix().OffDiagonalMoments().Z());
   EXPECT_FALSE(link.Inertial().MassMatrix().IsValid());
 }
+
+/////////////////////////////////////////////////
+TEST(DOMLink, ValidInertialPoseRelTo)
+{
+  std::ostringstream stream;
+  stream << "<?xml version=\"1.0\"?>"
+         << "<sdf version='1.7'>"
+         << "  <model name='A'>"
+         << "    <link name='B'>"
+         << "      <inertial>"
+         << "        <pose relative_to=''>0 0 0 0 0 0</pose>"
+         << "      </inertial>"
+         << "    </link>"
+         << "  </model>"
+         << "</sdf>";
+
+  sdf::SDFPtr sdf(new sdf::SDF());
+  sdf::init(sdf);
+  ASSERT_TRUE(sdf::readString(stream.str(), sdf));
+
+  sdf::Link link;
+  sdf::Errors errors =
+      link.Load(sdf->Root()->GetElement("model")->GetElement("link"));
+
+  EXPECT_TRUE(errors.empty());
+  EXPECT_EQ(link.RawPose(), ignition::math::Pose3d(0, 0, 0, 0, 0, 0));
+}
+
+/////////////////////////////////////////////////
+TEST(DOMLink, InvalidInertialPoseRelTo)
+{
+  std::ostringstream stream;
+  stream << "<?xml version=\"1.0\"?>"
+         << "<sdf version='1.7'>"
+         << "  <model name='A'>"
+         << "    <frame name='C'>"
+         << "      <pose>0 0 1 0 0 0</pose>"
+         << "    </frame>"
+         << "    <link name='B'>"
+         << "      <inertial>"
+         << "        <pose relative_to='C'>0 0 0 0 0 0</pose>"
```

Comment:
Can you use a non-zero (i.e non-default) pose to ensure the pose is parsed even if there is an error?

Reply:
78bd7730848e8c1c5fe9fd7d601003f97609333b, I also changed the test to compare the inertial pose (not the link pose).